qa/tasks/cephadm_cases/test_cli.py

```diff
@@ -44,30 +44,30 @@ def test_yaml(self):
 
     def test_pause(self):
         self._orch_cmd('pause')
-        self.wait_for_health('CEPHADM_PAUSED', 30)
+        self.wait_for_health('CEPHADM_PAUSED', 60)
```

Comment:
Maybe this is overkill, but perhaps it would be helpful to have a "constant" for the standard wait period? Something like `heath_wait_secs`  In other cases you can still have non-default values OR do a multiple of the `health_wait_secs` value...

Reply:
you're probably right here, although I didn't see the wait times as sort of connected in any way (it's just intervals like 30 and 60 are natural) so I didn't set anything up. I'm probably going to merge this since the qa run was good and I don't want to have to test it again right now but I'll keep the idea in mind if I look here again.